assets/misc/gold_seqMethod_to_nmdc_instrument_set.tsv

```diff
@@ -0,0 +1,11 @@
+#GOLD SeqMethod	NMDC instrument_set
```

Comment:
Why is there a `#` in the header line?

Reply:
makes it easier to skip header columns. I can remove if you don't want it.

Reply:
Can we remove the `#`? We should be only be using `\t` as the delimiter.